schema/deploy/tables/application_003.sql

```diff
@@ -0,0 +1,23 @@
+-- Deploy ggircs-portal:tables/application_003 to pg
+-- requires: tables/application_002
+
+begin;
+
+alter table ggircs_portal.application add column swrs_report_id integer;
+create unique index application_swrs_report_id_uindex on ggircs_portal.application(swrs_report_id);
+comment on column ggircs_portal.application.swrs_report_id is 'The swrs id of the swrs report (swrs.report.swrs_report_id). This id is the id assigned by SWRS (Single Window Reporting System) when the report was filed federally. This id is different from report_id (swrs.report.id) as that ID is generated by our ETL process';
+
+with app_swrs as (
+  select a.id, r.swrs_report_id
+    from swrs.report r
+    join ggircs_portal.facility f
+    on r.swrs_facility_id = f.swrs_facility_id
+    join ggircs_portal.application a
+    on f.id = a.facility_id
+    and a.reporting_year = r.reporting_period_duration
+)
+update ggircs_portal.application a_outer
+set swrs_report_id = (select swrs_report_id from app_swrs where id = a_outer.id)
+where a_outer.id = (select id from app_swrs where id = a_outer.id);
```

Comment:
This looks about right. I assume you tested this manually in your local environment?

Reply:
Yeah, I had to change some dates & stuff locally, but I managed to get it to update some existing applications (and not update others where the reporting year doesn't match)